Unable to run application on windows 8.1

I cloned the project and I am trying to run the app using meteor command. I am getting the error couldn't install npm packages from npm-shrinkwrap: Command failed: C:\Users\abdul raheem janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt os.windows.x86_32\dev_bundle\bin\npm.cmd

I am trying to run on windows 8.1 and my meteor version is 1.4.2

C:\Users\abdul raheem janjua\Rocket.Chat>meteor
[[[[[ C:\Users\abdul raheem janjua\Rocket.Chat ]]]]]

=> Started proxy.
=> Started MongoDB.
rocketchat:cas: updating npm dependencies -- cas...
rocketchat:oembed: updating npm dependencies -- he, jschardet, iconv-lite,
ip-range-check...
rocketchat:slackbridge: updating npm dependencies -- slack-client...
rocketchat:ui-master: updating npm dependencies -- clipboard...
rocketchat:tutum: updating npm dependencies -- redis...
rocketchat:ui-sidenav: updating npm dependencies -- less,
less-plugin-autoprefix...
steffo:meteor-accounts-saml: updating npm dependencies -- xml2js,
xml-crypto, xmldom, connect, xmlbuilder, querystring, xml-encryption...
=> Errors prevented startup:

   While building package rocketchat:cas:
   error: couldn't install npm packages from npm-shrinkwrap: Command
   failed: C:\Users\abdul raheem
   janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt-os.windows.x86_32\
dev_bundle\bin\npm.cmd
   install
   npm ERR! git clone --template=C:\Users\abdul raheem
   janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt-os.windows.x86_32\
dev_bundle\.npm\_git-remotes\_templates
   --mirror https://github.com/kcbanner/node-cas.git C:\Users\abdul raheem
   janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt-os.windows.x86_32\
dev_bundle\.npm\_git-remotes\git-https-github-com-kcbanner-node-cas-git-fcd27dad
333223b3b75a048bce27973fb3ca0f62-4bf73491:
   undefined
   npm ERR! git clone --template=C:\Users\abdul raheem
   janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt-os.windows.x86_32\
dev_bundle\.npm\_git-remotes\_templates
   --mirror https://github.com/kcbanner/node-cas.git C:\Users\abdul raheem
   janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt-os.windows.x86_32\
dev_bundle\.npm\_git-remotes\git-https-github-com-kcbanner-node-cas-git-fcd27dad
333223b3b75a048bce27973fb3ca0f62-4bf73491:
   undefined
   npm ERR! Windows_NT 6.3.9600
   npm ERR! argv "C:\\Users\\abdul raheem
   janjua\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.2\\mt-os.windows.
x86_32\\dev_bundle\\bin\\node.exe"
   "C:\\Users\\abdul raheem
   janjua\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.2\\mt-os.windows.
x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"
   "install"
   npm ERR! node v4.6.1
   npm ERR! npm  v3.10.9
   npm ERR! code ENOGIT

   npm ERR! not found: git
   npm ERR!
   npm ERR! Failed using git.
   npm ERR! This is most likely not a problem with npm itself.
   npm ERR! Please check if you have git installed and in your PATH.

   npm ERR! Please include the following file with any support request:
   npm ERR!     C:\Users\abdul raheem
   janjua\Rocket.Chat\packages\rocketchat-cas\.npm\package-new-1o87e07\npm-debug
.log
   npm ERR! git clone --template=C:\Users\abdul raheem
   janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt-os.windows.x86_32\
dev_bundle\.npm\_git-remotes\_templates
   --mirror https://github.com/kcbanner/node-cas.git C:\Users\abdul raheem
   janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt-os.windows.x86_32\
dev_bundle\.npm\_git-remotes\git-https-github-com-kcbanner-node-cas-git-fcd27dad
333223b3b75a048bce27973fb3ca0f62-4bf73491:
   undefined
   npm ERR! git clone --template=C:\Users\abdul raheem
   janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt-os.windows.x86_32\
dev_bundle\.npm\_git-remotes\_templates
   --mirror https://github.com/kcbanner/node-cas.git C:\Users\abdul raheem
   janjua\AppData\Local\.meteor\packages\meteor-tool\1.4.2\mt-os.windows.x86_32\
dev_bundle\.npm\_git-remotes\git-https-github-com-kcbanner-node-cas-git-fcd27dad
333223b3b75a048bce27973fb3ca0f62-4bf73491:
   undefined
   npm ERR! Windows_NT 6.3.9600
   npm ERR! argv "C:\\Users\\abdul raheem
   janjua\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.2\\mt-os.windows.
x86_32\\dev_bundle\\bin\\node.exe"
   "C:\\Users\\abdul raheem
   janjua\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.2\\mt-os.windows.
x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js"
   "install"
   npm ERR! node v4.6.1
   npm ERR! npm  v3.10.9
   npm ERR! code ENOGIT

   npm ERR! not found: git
   npm ERR!
   npm ERR! Failed using git.
   npm ERR! This is most likely not a problem with npm itself.
   npm ERR! Please check if you have git installed and in your PATH.

   npm ERR! Please include the following file with any support request:
   npm ERR!     C:\Users\abdul raheem
   janjua\Rocket.Chat\packages\rocketchat-cas\.npm\package-new-1o87e07\npm-debug
.log


=> Your application has errors. Waiting for file change.
Terminate batch job (Y/N)?
^C
C:\Users\abdul raheem janjua\Rocket.Chat>ulimit -n
'ulimit' is not recognized as an internal or external command,
operable program or batch file.

--- a/packages/rocketchat-cas/package.js
+++ b/packages/rocketchat-cas/package.js
@@ -27,5 +27,5 @@ Package.onUse(function(api) {
 });
 
 Npm.depends({
-	cas: 'git+https://github.com/kcbanner/node-cas#fcd27dad333223b3b75a048bce27973fb3ca0f62'
+	cas: 'https://github.com/kcbanner/node-cas/tarball/fcd27dad333223b3b75a048bce27973fb3ca0f62'
 });

--- a/packages/rocketchat-i18n/package.js
+++ b/packages/rocketchat-i18n/package.js
@@ -9,8 +9,9 @@ Package.onUse(function(api) {
 	api.use('templating', 'client');
 
 	var fs = Npm.require('fs');
-	fs.readdirSync(process.env.PWD + '/packages/rocketchat-i18n/i18n').forEach(function(filename) {
-		if (filename.indexOf('.json') > -1 && fs.statSync(process.env.PWD + '/packages/rocketchat-i18n/i18n/' + filename).size > 16) {
+	var workingDir = process.env.PWD || '.';
+	fs.readdirSync(workingDir + '/packages/rocketchat-i18n/i18n').forEach(function(filename) {
+		if (filename.indexOf('.json') > -1 && fs.statSync(workingDir + '/packages/rocketchat-i18n/i18n/' + filename).size > 16) {
 			api.addFiles('i18n/' + filename);
 		}
 	});

